File: GFG/Array/Day4/FrequenciesInALimitedArray.java
import java.util.*;

public class FrequenciesInALimitedArray{
    public List<Integer> frequencyCount(int[] arr) {
        // code here
        HashMap<Integer,Integer> map=new HashMap<>();
        
        for(int n:arr){
            if(map.containsKey(n)){
                map.put(n,map.get(n)+1);
                continue;
            }
            map.put(n,1);
        }
        
        ArrayList<Integer> res=new ArrayList<>();
        
        for(int i=1;i<=arr.length;i++){
            if(map.containsKey(i)){
                res.add(map.get(i));
                continue;
            }
            res.add(0);
        }
        
        return res;
    }
}

File: GFG/Array/Day4/FrequenciesInALimitedArray.py
class Solution:
    def frequencyCount(self, arr):
        #  code here
        map={}
        res=[]
        
        for num in arr:
            if num in map:
                map[num]=map[num]+1
            else:
                map[num]=1
                
        for i in range(1,len(arr)+1):
            if i in map:
                res.append(map[i])
            else:
                res.append(0)
                
        return res